packages/create-webpack-app/src/generators/init/svelte.ts

```diff
@@ -0,0 +1,221 @@
+import { NodePlopAPI, Answers, ActionType } from "../../types";
+import { dirname, resolve, join } from "path";
+import ejs from "ejs";
+import { DynamicActionsFunction } from "node-plop";
+import { fileURLToPath } from "url";
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+export default async function (plop: NodePlopAPI) {
+  // dependencies to be installed
+  const dependencies: Array<string> = [
+    "webpack",
+    "webpack-cli",
+    "svelte",
+    "svelte-loader",
+    "webpack-dev-server",
+    "html-webpack-plugin",
+  ];
+
+  await plop.load("../../utils/pkgInstallAction.js", {}, true);
+
+  plop.setDefaultInclude({ generators: true, actionTypes: true });
+  plop.setPlopfilePath(resolve(__dirname, "../../plopfile.js"));
+
+  // Define a base generator for the Svelte project structure
+  plop.setGenerator("init-svelte", {
+    description: "Create a basic Svelte-webpack project",
+    prompts: [
+      {
+        type: "input",
+        name: "projectPath",
+        message: "Enter the project destination:",
+        default: ".",
+        filter: (input) => {
+          return resolve(process.cwd(), input);
+        },
+      },
+      {
+        type: "list",
+        name: "langType",
+        message: "Which of the following JS solutions do you want to use?",
+        choices: ["ES6", "Typescript"],
+        default: "ES6",
+      },
+      {
+        type: "confirm",
+        name: "useSvelteStore",
+        message: "Do you want to use Svelte store for state management?",
```

Comment:
Okay will work on this soon, So does this mean in react I have to do the same since useState comes with react.

Can i do it in such a way that the default answer is true instead of false like in new react generator #4242?
this way it will be included in default setup.

Reply:
yeah, useState should be included by default.